resources/js/Pages/Dashboard.vue

```diff
@@ -63,20 +63,28 @@ import { ref } from 'vue';
                             </div>
                             <div class="post_comment-box row g-3">
                                 <div class="col-md-8">
+                                    <InputLabel class="d-none" :for="'new-comment_' + index"  value="Comment here"/>
                                     <TextInput
                                         type="text"
                                         placeholder="comment here"
+                                        :id="'new-comment_' + index"
                                         v-model="comments[index]"
+                                        aria-label="comment input field"
                                     />
                                 </div>
                                 <PrimaryButton @click="postComment(comments[index], post.id)" class="col-md-4">
                                     post comment
                                 </PrimaryButton>
                             </div>
                         </div><!-- post-item end -->
-                        <PrimaryButton v-if="this.posts.length>3 && this.postLength < this.posts.length" @click="loadMore" class="offset-lg-5 col-md-3">
-                            Load More
-                        </PrimaryButton>
+                        <!-- pagination -->
+                        <nav role="navigation" aria-label=”pagination”>
+                            <ul class="pagination float-end">
+                                <li v-for="(link,index) in loadMoreLinks"  class="page-item">
+                                    <a v-bind:class="{ active: link.active }" class="page-link" @click="loadMorePosts(link.url)" v-if="index>0 && index<loadMoreLinks.length-1" href="javascript:void(0)">{{ index }}</a>
```

Comment:
You don't need to type `v-bind`, `:class` will also work.

Reply:
Also, why have you used `javascript:void(0)`?

Reply:
this will get removed as I have already resolved these things in my earlier PR.